pymongo/asynchronous/encryption.py

```diff
@@ -836,7 +831,7 @@ async def encrypt(
             when the algorithm is :attr:`Algorithm.INDEXED`.  An integer value
             *must* be given when the :attr:`Algorithm.INDEXED` algorithm is
             used.
-        :param range_opts: Experimental only, not intended for public use.
+        :param range_opts: Index options for `range` queries.
```

Comment:
We need notes in the docstrings for the new public parameter being added.

Reply:
For `trimFactor`? Or for all of the `rangeOpts` themselves?

Reply:
`rangeOpts`

Reply:
Done.